src/components/TextInput/BaseTextInput.js

```diff
@@ -304,7 +304,9 @@ class BaseTextInput extends Component {
                 {this.props.autoGrow && (
                     <Text
                         style={[...this.props.inputStyle, styles.hiddenElementOutsideOfWindow, styles.visibilityHidden]}
-                        onLayout={e => this.setState({textInputWidth: e.nativeEvent.layout.width})}
+                        onLayout={e => this.setState({textInputWidth: e.nativeEvent.layout.width + 2})}
+
+                        // Added +2 in width so that the first digit of split amount do not cut off on mWeb.
```

Comment:
```suggestion
                        // Add +2 to width so that the first digit of amount do not cut off on mWeb - https://github.com/Expensify/App/issues/8158
```